predtimechart-config.yml

```diff
@@ -1,12 +1,11 @@
 ---
 rounds_idx: 0
-model_tasks_idx: 1
+model_tasks_idx: 0
 reference_date_col_name: 'reference_date'
 target_date_col_name: 'target_end_date'
 horizon_col_name: 'horizon'
-initial_checked_models: ['UMass-flusion', 'UMass-trends_ensemble']
-disclaimer: "This is a disclaimer. It presents important information for your stakeholders."
-target_data_file_name: 'target-hospital-admissions.csv'
+initial_checked_models: ['CovidHub-ensemble', 'CovidHub-baseline']
```

Comment:
Checked that the models/target data in lines 7-8 exist over in the corresponding repo 👍